ptbrowne/tcpasyncio/tcp_hello_client.py

```diff
@@ -0,0 +1,39 @@
+import asyncio
+
+SERVERS = {
+    'en': 8888,
+    'fr': 8889
+}
+
+WHATSYOURNAME = {
+    'en': 'What\'s your name ?',
+    'fr': 'Quel est ton nom ?'
+}
+
+async def tcp_echo_client(language, name):
+    loop = asyncio.get_event_loop()
+    reader, writer = await asyncio.open_connection(
+        '127.0.0.1',
+        SERVERS[language],
+        loop=loop
```

Comment:
```suggestion
        loop=asyncio.get_event_loop()
```
If you inline the local 'loop' variable, then this clarifies that the var is not used anywhere else. The reader can kick it out of its cognitive context. It also works well here, where the name of the var does not add much more meaning to the right-hand side expression that is pretty clear.